src/test/java/core/basesyntax/service/impl/DataValidatorServiceImplTest.java

```diff
@@ -31,9 +31,8 @@ public void validator_checkDataInput_BalanceOperation_PositiveQuality_ok() {
     @Test
     public void validator_checkDataInput_SupplyOperation_PositiveQuality_ok() {
         String[] data = new String[]{"s", "banana", "10"};
-        boolean expected = true;
         boolean actual = validator.checkDataInput(data);
-        Assert.assertEquals(expected, actual);
+        Assert.assertTrue(actual);
     }
 
     @Test
```

Comment:
you can use `assertTrue()/False()` in all methods where you compare with boolean